Use v7 sidekick (#173)
commerce-docs/microsite-commerce-storefront: src/content/docs/get-started/index.mdx

### Set up Sidekick for content

<Diagram caption="Set up Sidekick.">![Set up Sidekick.](images/setup-sidekick.webp)</Diagram>

<Callouts>

1. Install Sidekick from the Chrome Web Store and pin the Sidekick extension to your browser's toolbar.

> 1. Open your content folder and sign into the Sidekick.

Reviewer: ```suggestion
1. Open your content folder. You should now see an overlay menu bar at the bottom of your content pages.
```

> 1. Click the hamburger menu of the Sidekick and then **Add this project**.

Reviewer: ```suggestion
1. From the overlay menu, select **Sign-in**.
1. After you are signed in, select the **Options** icon (three horizontal lines ⫢) from the Sidekick menu and select **Add this project** from the options.
```

> 1. Return to your folder, select **ALL files**, select the **Sidekick** extension, and click the **Preview** button. Repeat this for all files in each folder and each subfolder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where they can be downloaded and displayed in your browser for local development.

Reviewer: ```suggestion
1. In your content folder, select all content files, then select the **Preview** button from the Sidekick overlay at the bottom of your browser page.  Repeat this for all files in each folder and each subfolder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where they can be downloaded and displayed in your browser for local development.
```
